Assets/Animation/Player/Player.controller

```diff
@@ -13,37 +13,37 @@ AnimatorController:
     m_DefaultFloat: 0
     m_DefaultInt: 0
     m_DefaultBool: 0
-    m_Controller: {fileID: 0}
+    m_Controller: {fileID: 9100000}
```

Comment:
What are all these meta files changing?

Reply:
This could be moving between 1.9f1 and 1.9f2, but it shouldn't cause any issue